src/Material_TEST.cc

```diff
@@ -30,6 +30,7 @@ TEST(DOMMaterial, Construction)
   EXPECT_EQ(ignition::math::Color(0, 0, 0, 1), material.Specular());
   EXPECT_EQ(ignition::math::Color(0, 0, 0, 1), material.Emissive());
   EXPECT_TRUE(material.Lighting());
+  EXPECT_EQ(0, material.RenderOrder());
```

Comment:
These are giving compiler warnings

```suggestion
  EXPECT_FLOAT_EQ(0.0, material.RenderOrder());
```